packages/contracts-bedrock/test/L1/ResourceMetering.t.sol

```diff
@@ -282,6 +282,8 @@ contract ArtifactResourceMetering_Test is Test {
     /// @dev Generates a CSV file. No more than the L1 block gas limit should
     ///      be supplied to the `meter` function to avoid long execution time.
     function test_meter_generateArtifact_succeeds() external {
+        vm.skip({ skipTest: true });
```

Comment:
is this temporary? We should add a todo if so.

Reply:
Good call @Inphi, i have added a comment describing why this was added